anyway/widgets/all_locations_widgets/killed_and_injured_count_per_age_group_widget_utils.py

```diff
@@ -148,3 +142,11 @@ def create_query_for_killed_and_injured_count_per_age_group(
             .order_by(asc(InvolvedMarkerView.age_group))
         )
         return query
+
+
+def adapt_location_fields_to_involve_table(filter: dict) -> dict:
+    res = copy.copy(filter)
+    for field in ["yishuv_name", "yishuv_symbol"]:
+        if field in res:
+            res[f"involve_{field}"] = res.pop(field)
```

Comment:
@ziv17 I think this should be:
`res[f"accident_{field}"] = res.pop(field)`
Since we're filtering by the accident location rather than the involved yishuv name / symbol

Reply:
I will change. Just note that it was `involved` since it moved from `road_segment_widget` to `all_location_widget` on December 2023.

Reply:
Thanks @ziv17 for the update. That's ok since we didn't insert this widget to prod yet.
Will do after this fix :)

Reply:
Fixed.

Reply:
Done.